xonsh/environ.py

```diff
@@ -1865,12 +1865,10 @@ def get_default(self, key, default=None):
             return default
 
     def get_docs(self, key, default=None):
-        """Gets the documentation for the environment variable."""
+        """Gets the documentation for the environment variable. May be None"""
         vd = self._vars.get(key, None)
         if vd is None:
-            if default is None:
-                default = Var()
-            return default
+            return None
```

Comment:
Shouldn't this return `default`?

Reply:
`get_docs()` needs to ensure returned *obj*.doc_default is `str`, need to fix up default.
But the latest commit is a cleaner way to handle this, and lets me to remove the extra code where invoked in xonsh/xonfig.py (and where it should have been in the 2 other invocations I missed on the first go-round).